do.sh

```diff
@@ -1,286 +1,295 @@
-set statusline+=%#StatuslineNumbers#   " Set color for line numbers
-set statusline+=%l,                    " Line number
-set statusline+=\                      " A space
-set statusline+=%-3c                    " Column number
-set statusline+=\ \|\                  " A separator
-set statusline+=%L                     " Total number of lines
-
-" Remove ugly Nerdtree statusline "
-let g:NERDTreeStatusline = '%#NonText#'
-
-" Rename tabs to show tab number.
-" (Based on http://stackoverflow.com/questions/5927952/whats-implementation-of-vims-default-tabline-function)
-if exists("+showtabline")
-    function! MyTabLine()
-        let s = ''
-        let wn = ''
-        let t = tabpagenr()
-        let i = 1
-        while i <= tabpagenr('$')
-            let buflist = tabpagebuflist(i)
-            let winnr = tabpagewinnr(i)
-            let s .= '%' . i . 'T'
-            let s .= (i == t ? '%1*' : '%2*')
-            let s .= ' '
-            let wn = tabpagewinnr(i,'$')
-
-            let s .= '%#TabNum#'
-            let s .= i
-            " let s .= '%*'
-            let s .= (i == t ? '%#TabLineSel#' : '%#TabLine#')
-            let bufnr = buflist[winnr - 1]
-            let file = bufname(bufnr)
-            let buftype = getbufvar(bufnr, 'buftype')
-            if buftype == 'nofile'
-                if file =~ '\/.'
-                    let file = substitute(file, '.*\/\ze.', '', '')
-                endif
-            else
-                let file = fnamemodify(file, ':p:t')
-            endif
-            if file == ''
-                let file = '[No Name]'
-            endif
-            let s .= ' ' . file . ' '
-            let i = i + 1
-        endwhile
-        let s .= '%T%#TabLineFill#%='
-        let s .= (tabpagenr('$') > 1 ? '%999XX' : 'X')
-        return s
-    endfunction
-    set stal=2
-    set tabline=%!MyTabLine()
-    set showtabline=1
-    highlight link TabNum Special
-endif
-EOM
+#!/bin/bash
+
+echo "begin install vim"
+
+# Option to install in Windows
+echo "Do you run this script on windows?([Y]es/[N]o)"
+read windows
+
+if [[ ($windows == "Y") ]]; then
+  choco install vim
+else
+  echo "Where do you want to install vim?"
+  echo "If the directory doesn't exist, vim will be installed in the given directory"
+  echo "If the directory exists a new folder will be created"
+  read location
+
+  # Get it
+  if [ ! -d "$location" ]; then
+    mkdir -p $location
+  else
+    location="$location/vim"
+  fi
+
+  git clone https://github.com/vim/vim.git $location
+  cd "$location/src"
+
+  ./configure \
+      --enable-pythoninterp \
+      --enable-cscope \
+      --with-features=huge \
+      --with-x
+
+  # Build and install
+  if ! make; then
+    echo ""
+    echo ""
+    echo "Couldn't compile vim. Make sure you have the necessary build tools"
+    echo ""
+    exit 1;
+  fi
+  sudo make install
+fi
+
+# Install LanguageTool in the same folder as vim

```

Comment:
Can you check why all these lines are being marked as modified?
Maybe you changed the line ending to windows style by mistake

Reply:
Is correct your comment, Windows changed the line ending. I working to fixed